src/styles/global.scss

```diff
@@ -0,0 +1,22 @@
+@import './variables';
+
+:root {
+  --primary-color: #{$primary-color};
+  --secondary-color: #{$secondary-color};
+  --text-color: #{$text-color};
```

Comment:
**praise:** nice job on using CSS variables. Note to take care to use them in your components if you really want to follow through (i have seen some hard-coded colors).